source/initial-dns-seedlist-discovery/tests/replica-set/dbname-with-commas.yml

```diff
@@ -0,0 +1,13 @@
+uri: "mongodb+srv://test1.test.build.10gen.cc/some,db?replicaSet=repl0"
```

Comment:
In python we parse this correctly but we also unescape the database name (using [unquote_plus](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.unquote_plus)). Should we test using an escaped version as well? Do you know if that quoting behavior is standard?

```
>>> parse_uri('mongodb+srv://test1.test.build.10gen.cc/some,db?replicaSet=repl0')['database']
'some,db'
>>> parse_uri('mongodb+srv://test1.test.build.10gen.cc/some%2Cdb?replicaSet=repl0')['database']
'some,db'
```

Reply:
I can add a test for the escaped version as well, since i think most drivers would handle escaped URIs.

Reply:
Reference PR to show this passes in Node, tests also added here.

https://github.com/mongodb/node-mongodb-native/pull/2844